docs/reference/cluster/nodes-reload-secure-settings.asciidoc

```diff
@@ -22,9 +16,41 @@ The first command reloads the keystore on each node. The seconds allows
 to selectively target `nodeId1` and `nodeId2`. The node selection options are
 detailed <<cluster-nodes,here>>.
 
-Note: It is an error if secure settings are inconsistent across the cluster
-nodes, yet this consistency is not enforced whatsoever. Hence, reloading specific
-nodes is not standard. It is only justifiable when retrying failed reload operations.
+NOTE: It is an error if secure settings are inconsistent across the cluster nodes, yet this consistency is not enforced
```

Comment:
```suggestion
NOTE: {es} returns an error if secure settings are inconsistent across the cluster nodes, but this consistency is not enforced
```

Reply:
We don't return an error ( this is what the "but this consistency is not enforced" is meant to explain ). I see how this is not very clear, I'm open to suggestions to rephrase this

Reply:
Thanks for the clarification. Here's a suggested rephrasing. 

```suggestion
NOTE: {es} requires consistent secure settings across the cluster nodes, but this consistency is not enforced.
```

Let me know if this is accurate. Also note that this removes the "whatsover." on the next line.